src/less/components/payex/toast.less

```diff
@@ -13,9 +13,9 @@
 		}
 	}
 
-	i {
-		&.material-icons {
-			font-family: "Material Icons";
-		}
+	// TODO: is it actually needed ? why ? is it to make sure it's material icons normal and NOT material icons OUTLINED ?
+	// maybe we can remove it
```

Comment:
I cannot see any reason for this rule, I suppose the it had something to do with the close icon. but as it is implemented I cant see it fixing anything. I vote remove.